unified_planning/test/anml/hydrone.anml

```diff
@@ -0,0 +1,145 @@
+type Location;
+constant integer distance(Location a, Location b);
+
+fluent boolean visited(Location a);
+fluent boolean robot_at(Location x);
+fluent integer [0, 300] battery_charge;
+fluent boolean moving;
+
+action move(Location from, Location destination)
+{
+  duration := distance(from, destination);
+
+  [start] robot_at(from);
+  [start] destination != from;
+  [start] distance(from, destination) > 0;
+  [start] not moving;
+  [start] battery_charge >= distance(from, destination);
+
+  [start] moving := true;
+  [start] robot_at(from) := false;
+  [end] battery_charge := battery_charge - distance(from, destination);
+  [end] robot_at(destination) := true;
+  [end] moving := false;
+  [end] visited(destination) := true;
+};
+
+instance Location start_point, a1, a2, b1, b2, b3, b4, b5, b6;
+
+[start] distance(start_point, b1) := 70;
+[start] distance(start_point, b2) := 2;
+[start] distance(start_point, b3) := 30;
+[start] distance(start_point, b6) := 200;
+[start] distance(start_point, start_point) := -1;
```

Comment:
ANML normally allows not specifying a value of a state variable/constant fluent. So this line should not be necessary.

In also explicitly support marking value a unknown (with an effect to the `UNKNOWN` value).
This is in my view a crucial feature to combine state variable and temporal planning, with the transition notation `[start,end] loc == kitchen :-> bedroom;`.